test-requirements.txt

```diff
@@ -0,0 +1,5 @@
+configparser
+flake8
+jsonschema
+pytest
+requests
```

Comment:
These should at least pin versions, and preferably use hashing to assure we get the same requirements each time.